psi4/__init__.py

```diff
@@ -100,5 +100,7 @@
 import sys
 if "@ENABLE_PCMSolver@".upper() in ["1", "ON", "YES", "TRUE", "Y"]:
     sys.path.insert(1, "@PCMSolver_PYMOD@")
+if "@ENABLE_cppe@".upper() in ["1", "ON", "YES", "TRUE", "Y"]:
+    sys.path.insert(1, "@cppe_PYMOD@")
```

Comment:
this `_PYMOD` variable is provided by you, akin to https://github.com/PCMSolver/pcmsolver/blob/master/cmake/PCMSolverConfig.cmake.in#L143. I don't think that's happening?

Reply:
Will be done in cppe 0.0.9.

Reply:
Done.